.github/workflows/pr.yml

```diff
@@ -38,5 +38,5 @@ jobs:
         name: Check spelling
         with:
           skip: "*.svg,*.js,*.map,*.css,*.scss"
-          ignore_words_list: "aks,atleast,cros,ddress,fiel,ist,nd,ot,pullrequest,ser,shttp,wast,fo,seldomly,delt,cruzer,plack,secur,te"
+          ignore_words_list: "aks,atleast,cros,ddress,fiel,ist,nd,ot,Parth,pullrequest,ser,shttp,wast,fo,seldomly,delt,cruzer,plack,secur,te"
```

Comment:
```suggestion
          ignore_words_list: "aks,atleast,cros,ddress,fiel,ist,nd,ot,parth,pullrequest,ser,shttp,wast,fo,seldomly,delt,cruzer,plack,secur,te"
```

Reply:
@kimsauce, I'll go ahead and commit this change to see if that gets the PR past the spellcheck step.